.github/scripts/stop_runner_service.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set +e
+set -x
+
+# Get the service name
+RUNNER_SERVICE=$(cat "${RUNNER_WORKSPACE}/../../.service")
```

Comment:
Is this just for AWS GPU runner? I do not recall the .service file on GCP A100 runner.

Reply:
Chat With @weiwangmeta and confirm that GCP A100 runner has the same setup and `.service` file under the runner home directory